server/chat.ts

```diff
@@ -1115,6 +1115,13 @@ export class CommandContext extends MessageContext {
 					targetUser.id !== user.id) {
 					throw new Chat.ErrorMessage(this.tr`You are blocking private messages right now.`);
 				}
+				if (
+					user.settings.blockChallenges &&
+					(user.settings.blockChallenges === true || !Users.globalAuth.atLeast(targetUser, user.settings.blockChallenges)) &&
+					!targetUser.can('bypassblocks')
+				) {
+					throw new Chat.ErrorMessage(this.tr`You are blocking challenges right now.`);
+				}
```

Comment:
What is this code for? It seems like it prevents you from PMing users who can't challenge you?

Reply:
I don't know why anymore. But I think this code is useless.

Reply:
Remove it, then.